packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/initialize.st

```diff
@@ -18,6 +17,8 @@ initialize
 	fillingMorph := (((CircleMorph new) color: Color random) center: self center) borderWidth: 0.
 	
 	self backupLevel: self level.
+	
+	self lineWidth: STWSimulation normalLineWidth.
```

Comment:
no, there are dependencies. for the line width setter there needs to be a filling morph